### DISCUSSION TITLE
Marketplace components

### DISCUSSION THREAD

**Discussion by egas-tt** [microsoft/AzUrlShortener#428] [Q&A] [answered]
Hello everyone!

I am really interested in this app that you all created! I would like to test it out using my company's Azure subscription, however we have an internal policy that prevents us from using any components from Azure Marketplace as we are not set up for the billing for these items with our Enterprise Agreement.

I observed that in the Deployment guide (https://github.com/FBoucher/TinyBlazorAdmin/blob/main/deployment.md#2--deploy-azurlshortener-the-backend) the image shows a Marketplace checkbox. Can you confirm whether or not this app uses any components from the Marketplace?

Thank you

**Comment by fboucher** [comment, marked as answer]
There are no Marketplace components this is a 100% open-source solution that you deploy in your Azure Subscription. Once deployed it's just like any other resources in Azure. And in term of cost... I spend less then CAN$ 1 by month for mine.
